package.json

```diff
@@ -39,6 +39,7 @@
     "*.css"
   ],
   "devDependencies": {
+    "babel-polyfill": "^6.26.0",
```

Comment:
Can we only Polyfill `Promise` please? We do not want to blindly polyfill any current and future ES improvement (yet).